RecoTracker/Configuration/python/customizeMinPtForHitRecoveryInGluedDet.py

```diff
@@ -13,4 +13,14 @@ def customizeHitRecoveryInGluedDetOn(process):
        process.Chi2MeasurementEstimatorForP5.MinPtForHitRecoveryInGluedDet = 100000
    return process
 
+def customizeHitRecoveryInGluedDetTkSeedsOnly(process):
+   process = customizeMinPtForHitRecoveryInGluedDet(process,0.9)
+   if hasattr(process, "Chi2MeasurementEstimatorForP5"): # keep disabled for cosmics
+       process.Chi2MeasurementEstimatorForP5.MinPtForHitRecoveryInGluedDet = 100000
+   for X in "muonSeededMeasurementEstimatorForOutIn", "hitCollectorForOutInMuonSeeds":
```

Comment:
also, I do not see the change in hitCollectorForOutInMuonSeeds in #17807 
did the implementation change or was it missed in #17807 ?